code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -228,13 +231,15 @@
 
 	stat = CONSCIOUS
 	icon_state = "[initial(icon_state)]"
+	worn_icon_state = "[initial(icon_state)]"
```

Comment:
Think you could use `base_icon_state` instead?

Reply:
Fair enough!